content/guides/merchant-integration-qr-code-flow.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: QR Code Flow for Merchants
+description: How merchants can accept payments by presenting a QR code to the patron.
```

Comment:
@Wulingoh There's a mix of `Patron` and `patron` in these descriptions. We should aim to be consistent with all our other guides 👍